wandb/cli/cli.py

```diff
@@ -886,7 +897,8 @@ def _parse_settings(settings):
                 "queue": queue,
                 "run_spec": json.dumps(
                     construct_launch_spec(
-                        os.getcwd(),  # uri,
+                        os.environ.get(wandb.env.DIR, os.getcwd()),  # uri
```

Comment:
This is the URI to refer to the Scheduler process right? Should it be the local path? The scheduler could be run anywhere, I wonder if we should this some constant value that launch could ignore a lot of the LaunchProject set up for?

Reply:
Yes, this is for the Scheduler. This is what I want to eventually replace with a wandb maintained docker container. Put it into  https://hub.docker.com/u/wandb

Reply:
Im mostly just concerned because this URI isnt meaningul right? Like what if the launch agent is running else where? Can we make an escape hatch URI specific to the scheduler?

Reply:
Correct: it is really only there because launch requires a URI. I will replace it with a placeholder for now until we have a container job specific to the scheduler.